src/_includes/section-menu/contribute.html

```diff
@@ -33,6 +33,12 @@
     <li><a href="{{ site.baseurl }}/contribute/dependencies/">Dependencies guide</a></li>
   </ul>
 </li>
+<li>
+  <span class="section-nav-list-title">Policies</span>
+  <ul class="section-nav-list">
+    <li><a href="{{ site.baseurl }}/contribute/postcommits-policies/">Post-commit tests policies</a></li>
```

Comment:
There are 3 new documents but only one is listed here. Did you mean to list the other 2?

Reply:
I intended to add only 1. I don't believe them as big for having an entry in main navigation section.